tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -909,3 +909,67 @@ def softmax(x):
             result.append(DetectionPrediction(identifier, labels, det_scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return result
+
+
+class UltraLightweightFaceDetectionAdapter(Adapter):
+    """
+    Class for converting output of Ultra-Lightweight Face Detection models to DetectionPrediction representation
+    """
+    __provider__ = 'ultra_lightweight_face_detection'
+
+    @classmethod
+    def validate_config(cls, config, fetch_only=False, **kwargs):
+        return super().validate_config(
+            config, fetch_only=fetch_only, on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT
+        )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'scores_out': StringField(description="Scores output layer name."),
+            'boxes_out': StringField(description="Boxes output layer name."),
+            'score_threshold': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.7, optional=True,
+                description='Minimal accepted score for valid boxes'),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+        self.score_threshold = self.get_value_from_config('score_threshold')
+
+    def process(self, raw, identifiers, frame_meta):
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+
+        batch_scores = raw_outputs[self.scores_out]
+        batch_boxes = raw_outputs[self.boxes_out]
+
+        result = []
+        for identifier, scores, boxes in zip(identifiers, batch_scores, batch_boxes):
+            detections = {'labels': [], 'scores': [], 'x_mins': [], 'y_mins': [], 'x_maxs': [], 'y_maxs': []}
+
+            score = np.transpose(scores)[1]
+            mask = score > self.score_threshold
+            filtered_boxes, filtered_score = boxes[mask, :], score[mask]
+            if filtered_score.size != 0:
+                x_mins, y_mins, x_maxs, y_maxs = filtered_boxes.T
+
+                labels = np.full_like(filtered_score, 1, dtype=int)
+                detections['labels'].extend(labels)
+                detections['scores'].extend(filtered_score)
+                detections['x_mins'].extend(x_mins)
+                detections['y_mins'].extend(y_mins)
+                detections['x_maxs'].extend(x_maxs)
+                detections['y_maxs'].extend(y_maxs)
```

Comment:
why do you need this, you can simplify to return just arrays without storing them in dictionary

Reply:
Done